docs/api-inference/parameters.md

```diff
@@ -132,7 +132,7 @@ async function query(data) {
             headers: {
                 Authorization: `Bearer hf_***`,
                 "Content-Type": "application/json",
-                "x-wait-for-model": "true"
++               "x-wait-for-model": "true"
```

Comment:
Make sure to make type a `diff`

Reply:
https://github.com/huggingface/hub-docs/pull/1397/commits/8186c69306b90a869eff1c0cda5d42d88741e19a